src/main/java/io/github/thebusybiscuit/slimefun4/core/networks/cargo/ItemStackAndInteger.java

```diff
@@ -12,7 +12,14 @@
 
     ItemStackAndInteger(ItemStack item, int amount) {
         this.number = amount;
-        this.item = item;
+        if (item instanceof ItemStackWrapper) {
+            this.item = new ItemStack(item.getType(), item.getAmount());
+            if (item.hasItemMeta()) {
+                this.item.setItemMeta(item.getItemMeta());
+            }
+        } else {
```

Comment:
Not quite sure why this is done here...

Reply:
Depends, is `getItem()` guaranteed to be called? I would assume so, in which case a lazy computation would be a bit nonsencial. Otherwise I would be completely for the lazy route.

Reply:
I can't say on the top of my head if it's guaranteed; however, intuitively it seems so given that the wrapper is lazy. I'll have a look at the current usages and make changes if necessary.